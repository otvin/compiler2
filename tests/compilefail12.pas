program compilefail12(output);
{semicolon missing after a procedure definition}
var i:integer;
function one(a:integer):integer;
begin
	one := a + 1;
end;

procedure two(var b:integer)	
begin
	b := b + 2;
end;

begin
	i := 7;
	two (i);
	writeln(one(i))	
end.
