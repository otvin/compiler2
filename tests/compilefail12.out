Semicolon expected in file:tests/compilefail12.pas line:10    column:1  
